package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc && vite build",
+    "build:dev": "tsc -W",
```

Comment:
¿Para qué habéis añadido este script?

Reply:
para tener el compilardor en modo watch

Reply:
Pero estáis usando vite, que ya compila en modo watch.

Reply:
pero si hay error de typescript no se queja

Reply:
No sé si me mola que tengáis que estar tirando dos compiladores a la vez. No lo he necesitado nunca, con Vite.

Reply:
Pero eso no es un flujo normal. La mayor parte del tiempo vas a estar tocando TS, y con Vite deberías estar usando el compilador de Vite.

Reply:
jmmm